p4include/v1model.p4

```diff
@@ -270,7 +276,7 @@ extern register<T> {
      *
      * allocates storage for 512 values, each with type bit<32>.
      */
-    register(bit<32> size);
+    register(bit<32> size);  // FIXME -- arg should be `int` but that breaks typechecking
```

Comment:
please file an issue about this; probably a very small reproduction will be enough